.enlintrc.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
### Good work creating a new repository for the front end. 

*There are some things that you need to do in the way to connect both applications, front-end, and back-end.*

**Kindly as in the requirements is requested. Follow the steps to complete the front-end connection**

- Create the App component with react-router.
- Create the Greeting component that will display a greeting. Set it up as a route in your App component.
- Create a store, an action, and a reducer that will connect to your API endpoint to get the random greeting.
- Display the random greeting in your Greeting component.